katdal/chunkstore_s3.py

```diff
@@ -62,68 +116,55 @@ class S3ChunkStore(ChunkStore):
 
     Parameters
     ----------
-    client : :class:`botocore.client.S3` object
-        Pre-configured botocore S3 client
+    session : :class:`requests.Session` object
```

Comment:
Is this the same as a session factory or is this out of date?

Reply:
Was out of date - fixed.